izug/refegovservice/upgrades/20191031154412_archetypes_to_dexterity/types/izug.refegovservice.egovservice.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0"?>
+<object xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+    name="izug.refegovservice.egovservice"
+    meta_type="Dexterity FTI"
+    i18n:domain="izug.refegovservice">
+
+  <!-- Basic properties -->
+  <property
+      i18n:translate=""
+      name="title">EGov Service</property>
+  <property
+      i18n:translate=""
+      name="description">Description of an Government Service</property>
+
+  <property name="allow_discussion">False</property>
+  <property name="icon_expr"></property>
+
+  <!-- Hierarchy control -->
+  <property name="global_allow">True</property>
+  <!-- Schema, class and security -->
+  <property name="add_permission">izug.refegovservice.AddEGovService</property>
+  <property name="klass">izug.refegovservice.content.e_gov_service.EGovService</property>
+  <property name="schema">izug.refegovservice.content.e_gov_service.IEGovServiceSchema</property>
+
+  <!-- Enabled behaviors -->
+  <property name="behaviors" purge="false">
+
+    <element value="plone.app.content.interfaces.INameFromTitle"/>
+    <element value="plone.app.dexterity.behaviors.metadata.IBasic"/>
```

Comment:
This line is not in the copy in the default profile. Make sure that the upgrade step is in sync with the default profile state.

Reply:
✅